src/main/java/edu/kis/powp/command/ComplexCommand.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.powp.command;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ComplexCommand implements IDriverCommand {
+
+    private List<IDriverCommand> queue;
+
+    private ComplexCommand(Builder builder) {
+        this.queue = builder.queue;
+    }
+
+    @Override
+    public void execute() {
+        for (IDriverCommand command : queue) {
+            command.execute();
+        }
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static final class Builder {
+        private ArrayList<IDriverCommand> queue;
```

Comment:
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.